lib/Bio/EnsEMBL/DataCheck/Test/DataCheck.pm

```diff
@@ -276,7 +276,6 @@ sub row_subtotals {
 
   $dbc2 = $dbc1 if ! defined $dbc2;
   $sql2 = $sql1 if ! defined $sql2;
-  $min_proportion = 1 if ! defined $min_proportion;
```

Comment:
Was it just unused portion of code?

Reply:
This was a bug - the behaviour should be the same as that of `row_totals`, where if the value is undefined, we seek an exact match. This line meant that the value was always defined, and it was impossible toy use the function to test for exact matches.